modules/configuration/php/configuration.class.inc

```diff
@@ -88,6 +88,14 @@ class Configuration extends \NDB_Form
             'emergency' => 'Emergency',
         ];
 
+        $this->tpl_data['password_algos'] = [
```

Comment:
Could you use https://www.php.net/manual/en/function.password-algos.php (and add a "Default" option)

I'm not really sure what the point of having the blank option here is.

Reply:
Sure!

Reply:
That function doesn't output usable strings for the password_hash function unfortunately. It gives values such as: `'2y','argon2i, 'argon2id'`, but password_hash uses, 'PASSWORD_DEFAULT','PASSWORD_BCRYPT', 'PASSWORD_ARGON2I', and 'PASSWORD_ARGON2ID'

Reply:
PASSWORD_DEFAULT/PASSWORD_BCRYPT/etc are just built in constants with those values.

```
$ php -a
Interactive shell

php > print PASSWORD_DEFAULT;
2y
php > print PASSWORD_ARGON2I;
argon2i
php > print_r(password_algos());
Array
(
    [0] => 2y
    [1] => argon2i
    [2] => argon2id
)
php > print password_hash('test', 'argon2i');
$argon2i$v=19$m=65536,t=4,p=1$WlBObVNleHU0RHVCMG9DTw$3jZ/HN8N4r6YERoJJbMGup/d8rebjUZ72lYZcbTsjno
php > 
```

Reply:
oh sweet, thanks!